packages/atomic/vitest-utils/testing-helpers/fixtures/headless/search/summary-subcontroller.ts

```diff
@@ -0,0 +1,35 @@
+import type {QuerySummary, QuerySummaryState} from '@coveo/headless';
```

Comment:
You can name the file "summary-controller" instead; subcontrollers is a Headless commerce engine concept (there are no subcontrollers in Headless search).